.mk/development.mk

```diff
@@ -179,12 +179,15 @@ endif
 	@echo -e "\n==> Redeploying..."
 	kubectl rollout status -n $(OPERATOR_NS) --timeout=60s deployment netobserv-controller-manager
 	kubectl wait -n $(OPERATOR_NS) --timeout=60s --for condition=Available=True deployment netobserv-controller-manager
-	kubectl rollout status -n $(NAMESPACE) --timeout=60s deployment netobserv-plugin
```

Comment:
did u intentionally remove those 2 lines ?

Reply:
yes, since it assumes netobserv-plugin is already deployed, while csv could be patched before any plugin is deployed and there would no netobserv-plugin deployment to check rollout status on. This is in line with what's currently in there for ebpf/flp make targets

Reply:
I'm good with that, this wasn't really useful actually